src/in/2024-01/01/01.md

```diff
@@ -17,7 +17,9 @@ Kitab Mazmur telah ditulis dalam puisi Ibrani oleh penulis yang berbeda dari zam
 
 _*Pelajari pelajaran pekan ini untuk persiapan Sabat, 6 Januari._
 
-#### Bacaan Tambahan: Kutipan dari Ellen G. White
+---
+
+#### Additional Reading: Selected Quotes from Ellen G. White
```

Comment:
@PaulTitto If you want me to make this text "Bacaan Tambahan: Kutipan dari Ellen G. White" collapsable

Reply:
Yes sir, for the Indonesian version the text can be collapsed, Please. Thxu